### PR TITLE
finished "feature"

### PR DESCRIPTION
**Pull request by weissgilad**
- db/
    - added folder to contain all the lists etc that will be used in the future. current lists should be moved to here
    - added significant_words db, currently contains only words that can be repeated sarcastically

- util/
    - added TB_random.py. this file will contain our implementation of random stuff so we can better control how it behaves

- echo_filter.py
    - added sarcastic reply filter

- echo_functions.py
    - added sarcastic repeat. just adds ""

### CODE REVIEW

File: db/significant_words.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+words_to_repeat_sarcastically = [
```
Comment (neriat) [inline db/significant_words.py]: I think that the folder name "db" is a bit misleading. maybe constants or literals?

File: util/TB_random.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# this file exists so it will be simple to replace the randomness if needed
```
Comment (neriat) [inline util/TB_random.py]: file name should use lowercase only.

File: echo_filter.py [hunk 3]
```diff
@@ -1,11 +1,14 @@
-from random import random
+from random import random  # TODO replace this with tb_random
+from util.TB_random import tb_choice
```
Comment (neriat) [inline echo_filter.py]: imports should be grouped by: standard libs, 3rd parties and local (in that order)

File: echo_functions.py [hunk 4]
```diff
@@ -22,3 +22,11 @@ def email_action(raw_msg, name, response):
     text = "של מי" if is_empty else generate(name)
 
     response(text)
+
+
+def repeat_sarcastically_action(str, response):
```
Comment (neriat) [inline echo_functions.py]: naming a variable str will shadow builtin str. rename to string

File: echo_filter.py [hunk 5]
```diff
@@ -56,6 +59,18 @@ def generic_question(update: Update, context: CallbackContext) -> bool:
     return False
 
 
+def repeat_word_sarcastically_filter(update: Update, context: CallbackContext) -> bool:
```
Comment (neriat) [inline echo_filter.py:62]: we should decide if we want to use type hinting in function deceleration.
what do you think?

**Review by neriat — CHANGES_REQUESTED** [review]
i've left comments.
tell me what do you think

File: echo_filter.py (hunk 5 above)
Comment (weissgilad) [reply echo_filter.py:62]: python typing sucks really hard and i find it super annoying to keep track of. you can change it if you want

**Review by neriat — APPROVED** [review]

**Pull request merged**